.gitmodules

```diff
@@ -0,0 +1,8 @@
+# [submodule "mock-srv"]
+# 	path = ./mock-srv
+# 	url = https://github.tools.sap/I584814/cloud-extension-s4hana-business-process
+# 	branch = mock
+[submodule "mock-srv"]
+	path = mock-srv
+	url = ./
+	branch = mock
```

Comment:
submodule should point to -> https://github.com/SAP-samples/cloud-extension-ecc-business-process/tree/mock